CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+### v0.8.9
+This is a minor release of the spec which includes one addition to the spec and a utility function to translate between appc and golang architectures.
```

Comment:
"... spec which includes one additional isolator and..." to avoid repeating "spec".